icu4c/source/common/unicode/urename.h

```diff
@@ -1693,7 +1642,6 @@
 #define uscript_getCode U_ICU_ENTRY_POINT_RENAME(uscript_getCode)
 #define uscript_getName U_ICU_ENTRY_POINT_RENAME(uscript_getName)
 #define uscript_getSampleString U_ICU_ENTRY_POINT_RENAME(uscript_getSampleString)
-#define uscript_getSampleUnicodeString U_ICU_ENTRY_POINT_RENAME(uscript_getSampleUnicodeString)
```

Comment:
Removing this one definitely looks wrong. uscript.h has
```
U_COMMON_API icu::UnicodeString U_EXPORT2
uscript_getSampleUnicodeString(UScriptCode script);
```
with no recent changes.